modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/spds/aah_spds.h

```diff
@@ -0,0 +1,58 @@
+// SPDX-FileCopyrightText: 2024 The OpenSn Authors <https://open-sn.github.io/opensn/>
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/spds/spds.h"
+
+namespace opensn
+{
+
+class AAH_SPDS : public SPDS
+{
+public:
+  /**
+   * Creates a sweep-plane data structure (SPDS) for the given direction and grid.
+   *
+   * \param id The unique identifier for this SPDS.
+   * \param omega The angular direction for the sweep operation.
+   * \param grid The grid on which the sweep is performed.
+   * \param allow_cycles Whether cycles are allowed in the local and global swepp dependency graphs.
+   */
+  AAH_SPDS(int id, const Vector3& omega, const MeshContinuum& grid, bool allow_cycles);
+
+  /// Returns the id of this SPDS.
+  int GetId() { return id_; }
+
+  /// Return the levelize sweep global sweep TDG.
```

Comment:
I don't understand this.

Reply:
It should be:

`/// Return the levelized global sweep TDG.`

Fixed.